vcpkg.json

```diff
@@ -4,14 +4,34 @@
     "builtin-baseline": "f14984af3738e69f197bf0e647a8dca12de92996",
     "dependencies": [
         "bzip2",
+        "catch2",
+        "liblzma",
+        "libmagic",
+        "lz4",
         "openssl",
-        "zstd",
-        {
-            "name": "catch2",
-            "version>=": "3.1.0"
-        }
+        "spdlog",
+        "zlib",
+        "zstd"
```

Comment:
It would be interesting for another time to make compression filters an optional feature; I will later open a story.